### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ZeissS**
* Update http-proxy and node-etcd in package.json. 
* Optimized Dockerfile. 
* Added more logs for watcher.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "istanbul": "^0.2.6",
     "npmlog": "^0.0.6",
     "memcached": "^0.2.8",
-    "node-etcd": "^2.0.10",
+    "node-etcd": "^3.0.0",
```
Comment (teemow) [inline package.json]: devDependencies ^3.0.0 vs dependencies 3.0.2

**Comment by zyndiecate** [comment]
LGTM

**Comment by teemow** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (ZeissS) [inline package.json]: `^` means everything 3.x.x - which is the same as was there before.

Comment (ZeissS) [inline package.json]: with regards to 2.x.x and 2.0.10 that was there before ;)

**Pull request merged**